.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,36 +10,6 @@
 
 <!-- Link related issues: "Fixes #1234", "Closes #1234", or "Related to #1234" -->
```

Comment:
it'll make committers' lives easier if we make these regular text instead of comments

currently, people leave the comments in the description and then we have to manually go through and trim them out when committing the PR

if we make them regular text, then it'll be clear that they should overwrite/delete the prompts.

WDYT?

Reply:
sounds minor, but I probably spend like 30min a week doing this 😮‍💨